qutebrowser/mainwindow/statusbar/command.py

```diff
@@ -216,3 +221,13 @@ def sizeHint(self):
             text = 'x'
         width = self.fontMetrics().width(text)
         return QSize(width, height)
+
+    def incrementSearch(self):
```

Comment:
Various things to note here:

- Please add a `@pyqtSlot()` decorator to mark this as a slot (a method connected to the signal)
- The `textChanged` signal also gives you the new text, so you can use `@pyqtSlot(str)` and get a `text` argument, which makes this a bit easier.
- This should be called `increment_search` to follow [pep8](https://www.python.org/dev/peps/pep-0008/)
- However, I'd prever `incremental_search`
- However, this is "private", so it should be `_incremental_search` :wink: